courses/machine_learning/deepdive2/structured/solutions/3c_bqml_dnn_babyweight.ipynb

```diff
@@ -0,0 +1,1152 @@
+{
```

Comment:
<ul><li>Link &nbsp;<a href="https://github.com/GoogleCloudPlatform/training-data-analyst/raw/9d258e061481c537a601c44e6210d04af4f2f5ca/courses/machine_learning/deepdive2/structured/solutions/3a_bqml_baseline_babyweight.ipynb" target="_blank">BQML Baseline Model</a> not working</li><li>Link &nbsp;<a href="https://github.com/GoogleCloudPlatform/training-data-analyst/raw/9d258e061481c537a601c44e6210d04af4f2f5ca/courses/machine_learning/deepdive2/structured/solutions/3b_bqml_linear_transform_babyweight.ipynb" target="_blank">BQML Linear Models with Feature Engineering</a> not working</li><li>"We will create and evaluate a DNN model using BigQuery ML," -&gt; "We will"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Links don't work because previous labs aren't in this branch.</p><p>Ditto</p><p>Done</p>